config/localization/en/app.json

```diff
@@ -128,6 +131,7 @@
       "GO_TO_MARKET": "Go to market",
       "GOVERNANCE": "Governance",
       "HEDGIES": "Hedgies",
+      "HELP": "Help",
```

Comment:
This key exists already

Reply:
addressed in bb53067d74d4b1b48541e5e26847c81341be3130